src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1401,16 +1398,15 @@ private void ToggleTreeViewItemHighlighting (string path, bool isSelected)
             }
 
             // Next, deselect the parentnode in HelixWatch3DView
-
             var nodeGeometryModels = Model3DDictionary.Where(x => x.Key.Contains(nodePath) && x.Value is GeometryModel3D).ToArray();
-
             foreach (var nodeGeometryModel in nodeGeometryModels)
             {
                 nodeGeometryModel.Value.SetValue(AttachedProperties.ShowSelectedProperty, false);
             }
 
             // Then, select the individual node
-            var geometryModels = Model3DDictionary.Where(x => x.Key.Contains(path) && x.Value is GeometryModel3D).ToArray();
+            var pathInDict = path + ":";
+            var geometryModels = Model3DDictionary.Where(x => x.Key.Contains(pathInDict) && x.Value is GeometryModel3D).ToArray();
```

Comment:
@samuelcwl Thanks for fixing this! How about a simple `x.Key.StartsWith(path + ":")`? Just to make it similar to the one at line 1307 in the same file.